src/app/backend/handler/localehandler.go

```diff
@@ -96,8 +96,7 @@ func (handler *LocaleHandler) determineLocalizedDir(locale string) string {
 	for _, tag := range tags {
 		matchedLocale := ""
 		for _, l := range handler.SupportedLocales {
-			base, _ := tag.Base()
-			if l == base.String() {
+			if l == tag.String() {
```

Comment:
This comparison is case sensitive and according to http://www.metamodpro.com/browser-language-codes code for Chinese (Taiwan) is `zh-tw`. I'd prefer to keep it lower case just to be safe. 

Additionally without this change  if someone had browser locale set to `zh-xx` then it would match our `zh` category. With this change if i.e. someone will have `zh-hk` then it will fallback to english. I'd extend this:

1. Check for exact match
2. If exact match does not work then check base
3. If both checks fail fallback to english

Reply:
@floreks  Thanks you for your reply! These suggestions are very helpful to me, I can add this extension in the next commit that to make sure the comparison is safe.